documentation/asciidoc/topics/proc_get_cluster_credentials.adoc

```diff
@@ -1,70 +1,33 @@
+[id='getting_credentials-{context}']
+= Retrieving Cluster Credentials
 You can retrieve credentials for your {brandname} clusters as base64-encoded strings from secrets in your cluster namespace.
 
-[NOTE]
-====
-If you do not specify credentials when you create clusters, the {ispn_operator} automatically generates passwords and sets default usernames as follows:
+If you do not create secrets and credentials when you create clusters, the {ispn_operator} automatically generates them with the following defaults:
 
+Default usernames::
++
 * Management user is `admin`.
 * Application user is `developer`.
-====
 
-.Procedure
-. List the pods for the cluster, for example:
+Default credentials secrets::
 +
-ifndef::productized[]
-[source,options="nowrap",subs=attributes+]
-----
-include::cmd_examples/oc_get_pods.adoc[]
-----
-endif::productized[]
-ifdef::productized[]
-[source,options="nowrap",subs=attributes+]
-----
-include::cmd_examples/oc_get_rhdg_pods.adoc[]
-----
-endif::productized[]
-+
-. Get the secrets from one of the pods as `yaml` output, for example:
-+
-ifndef::productized[]
-[source,options="nowrap",subs=attributes+]
-----
-$ oc get pod/example-infinispan-0 -o yaml
-
-include::yaml_examples/credentials_ispn.yaml[]
-----
-endif::productized[]
-ifdef::productized[]
-[source,options="nowrap",subs=attributes+]
-----
-$ oc get pod/datagrid-minimal-0 -o yaml
+* `<pod_name>-mgmt-generated-secret` contains credentials for the management user.
```

Comment:
It's not `pod_name`, the generated secrets are not per pod. A single secret of each type is generated for the whole infinispan CRD instantation. So, the prefix would be `example-infinispan-mgmt...` or `example-infinispan-app-...`. I don't know how you're referencing to the infinispan CRD name in the docu.